### PR TITLE
Fix globalization issues

### PR DESCRIPTION
**Pull request by ajcvickers**
Issue #5765

### CODE REVIEW

**Comment by lajones** [comment]
Looks good. Rather than just the comment `// Interpolation OK...`, is there any better way of indicating that the code as written should be ignored as far as the globalization checks are concerned? An annotation perhaps (depends on what tool we run to detect the problem in the first place I guess)? If not then :shipit:

**Comment by ajcvickers** [comment]
@lajones I'll leave it to @bricelam to comment on what he is using and whether there is a way to suppress warnings.

**Comment by bricelam** [comment]
`#pragma disable` or `[SuppressMessage]` will work, but I don't think it's important until we actually automate the checks.

File: src/Microsoft.EntityFrameworkCore.Design.Core/Properties/DesignCoreStrings.resx [hunk 1]
```diff
@@ -255,4 +255,7 @@ Change your target project to the migrations project by using the Package Manage
   <data name="InvokeStartupMethodFailed" xml:space="preserve">
     <value>An error occurred while calling method '{method}' on startup class '{startupClass}'. Consider using IDbContextFactory to override the initialization of the DbContext at design-time. Error: {error}</value>
   </data>
+  <data name="CouldNotSerialize " xml:space="preserve">
```
Comment (Eilon) [inline src/Microsoft.EntityFrameworkCore.Design.Core/Properties/DesignCoreStrings.resx]: @bricelam there's a trailing space in the key here. Can we fix that?

Comment (Eilon) [inline src/Microsoft.EntityFrameworkCore.Design.Core/Properties/DesignCoreStrings.resx]: (I realize it probably has no effect, but just in case.)

**Pull request merged**